pkg/controller/oss/oss_controller.go

```diff
@@ -84,41 +76,16 @@ func (c *Connector) Connect(ctx context.Context, mg resource.Managed) (managed.E
 		return nil, errors.New(errNotBucket)
 	}
 
-	var (
-		secretKeySelector *xpv1.SecretKeySelector
-		region            string
-	)
-
 	if err := c.Usage.Track(ctx, mg); err != nil {
```

Comment:
We are already tracking usage in the new utility function `util.PrepareClient`. No need to it here anymore.

Reply:
Sorry, my bad. I've double-checked the rest and all are fine. Thanks.